src/apps/media/src/app-revamp/views/BinMedia.tsx

```diff
@@ -34,16 +40,62 @@ export const BinMedia = ({ addImagesCallback }: Props) => {
     skip: !isValidId,
   });
 
-  const { data: binGroups, isFetching: isGroupsFetching } =
+  const { data: unsortedBinGroups, isFetching: isGroupsFetching } =
     mediaManagerApi.useGetBinGroupsQuery(id, {
       skip: !isValidId,
     });
 
-  const { data: binFiles, isFetching: isFilesFetching } =
+  const { data: unsortedBinFiles, isFetching: isFilesFetching } =
     mediaManagerApi.useGetBinFilesQuery(id, {
       skip: !isValidId,
     });
 
+  const binFiles = useMemo(() => {
+    if (!unsortedBinFiles) return unsortedBinFiles;
+    switch (sortOrder) {
+      case "alphaAsc":
+        return [...unsortedBinFiles].sort((a, b) => {
+          return a.filename.localeCompare(b.filename);
+        });
+      case "alphaDesc":
+        return [...unsortedBinFiles].sort((a, b) => {
+          return b.filename.localeCompare(a.filename);
+        });
+      case "createdDesc":
+        return [...unsortedBinFiles].sort((a, b) => {
+          return (
+            new Date(b.created_at).getTime() - new Date(a.created_at).getTime()
+          );
+        });
+      // Default to API order
```

Comment:
I would prefer to explicitly sort it here rather than implicitly rely on the sort order from the `mediaManager` API.

Do you think I should remove the `createdDesc` case and just use the order from the `mediaManager` API?